packages/create-sitecore-jss/src/templates/nextjs-styleguide/src/components/fields/Styleguide-FieldUsage-Image.tsx

```diff
@@ -39,18 +40,24 @@ const StyleguideFieldUsageImage = (props: StyleguideFieldUsageImageProps): JSX.E
     />
 
     {/*
-      Srcset adaptive image usage example
-      Adaptive srcsets are supported using Sitecore server-side resizing.
-      The `srcSet` can use Sitecore image resizing parameters (i.e. w, h, mw, mh).
-      Sample create a srcset using two sizes (server resizing), 300 and 100px max widths, respecting aspect ratio.
-      IMPORTANT: srcSet params must be whitelisted for adaptive resizing to occur. See /sitecore/config/*.config (search for 'allowedMediaParams')
+      srcSet in Nextjs Image is set inside of the next.config by setting an array of deviceSizes inside the images option. See here: https://nextjs.org/docs/api-reference/next/image#device-sizes
+      These sizes should match your Sitecore server-side allowlist.
+      See /sitecore/config/*.config (search for 'allowedMediaParams')
+       image sizes are set similarly: https://nextjs.org/docs/api-reference/next/image#image-sizes
+
+      placeholder='blur' blurs the image while loading with a transparent pixel provided in blurDataUrl. You can also provide a color pixel for this functionality. See here: https://nextjs.org/docs/api-reference/next/image#placeholder
```

Comment:
Something is wrong with code formatting, try to use Prettier and split some lines, it's long